test/leiningen/monolith_test.clj

```diff
@@ -0,0 +1,65 @@
+(ns leiningen.monolith-test
```

Comment:
🤔 it might be good to put this under `lein-monolith` instead of mirroring the ns name here. I have a feeling leiningen is probably loading this namespace optimistically, thinking it might be a plugin or task code.

Reply:
Done.